frontend-react/e2e/spec/about-page.spec.ts

```diff
@@ -0,0 +1,196 @@
+            await aboutPage.page
+                .locator("div")
+                .filter({ hasText: /^Release notes$/ })
+                .getByRole("link")
+                .click();
+            await expect(aboutPage.page).toHaveURL(/.*about\/release-notes/);
+        });
+    });
+
+    test.describe("Values section", () => {
+        test("Accordion sections expand", async ({ aboutPage }) => {
+            const accordionCol = [
+                "We meet our partners where they are.",
+                "We are free to use.",
+                "We put better faster data into the hands of public health entities.",
+                "We reduce disparities in healthcare monitoring and surveillance.",
+            ];
+
+            for (let i = 0; i < accordionCol.length; i++) {
+                const accordionItem = `accordionItem_${i + 1}--content`;
+                await expect(
+                    aboutPage.page.getByTestId(accordionItem),
+                ).toBeHidden();
+
+                await aboutPage.page
+                    .getByRole("button", {
+                        name: accordionCol[i],
+                    })
+                    .click();
+
+                await expect(
+                    aboutPage.page.getByTestId(accordionItem),
+                ).toBeVisible();
+
+                await aboutPage.page
+                    .getByRole("button", {
+                        name: accordionCol[i],
+                    })
+                    .click();
+
+                await expect(
+                    aboutPage.page.getByTestId(accordionItem),
+                ).toBeHidden();
+            }
+        });
+    });
+
+    test.describe("Recommended Resources", () => {
+        test("Card navigation", async ({ aboutPage }) => {
+            const cardLinks = [
+                {
+                    name: "ReportStream overview PDF",
+                    url: "",
+                },
+                {
+                    name: "ReportStream API",
+                    url: "/developer-resources/api",
+                },
+                {
+                    name: "Release notes",
+                    url: "/about/release-notes",
+                },
+            ];
+
+            for (const cardLink of cardLinks) {
+                await aboutPage.page
+                    .getByTestId("CardGroup")
+                    .getByRole("link", { name: cardLink.name })
+                    .click();
+
+                if (cardLink.url.length) {
+                    await expect(aboutPage.page).toHaveURL(cardLink.url);
+                }
+
+                await aboutPage.page.goto(URL_ABOUT, {
+                    waitUntil: "domcontentloaded",
+                });
+            }
+        });
+    });
+
+    test.describe("Footer", () => {
+        test("has footer", async ({ aboutPage }) => {
+            await expect(aboutPage.page.locator("footer")).toBeAttached();
```

Comment:
The footer locator should be added to the BasePage class so it's available on every page. The scrolling utilities should likely also be integrated into BasePage. Small nits, however, and not critical blockers on overall review.